.circleci/smoketest_k8s.sh

```diff
@@ -53,7 +69,9 @@ kubectl create configmap scalyr-config \
 # Build local image (add .ci.k8s to version)
 TEMP_DIRECTORY=~/temp_directory
 mkdir $TEMP_DIRECTORY
+/bin/rm -rf $TEMP_DIRECTORY/*
```

Comment:
Code smell:  Never `rm -rf` a variable without checking whether its empty, and surrounding it in quotes during the `rm -rf`

It's not a problem in this instance because `$TEMP_DIRECTORY` is guaranteed to be `~/temp_directory`, *but* if for whatever reason some future update changes this or introduces a bug such that `$TEMP_DIRECTORY` is empty, the side effect shouldn't be `rm -rf /*`.  See for example [here](https://www.extremetech.com/extreme/197686-nasty-steam-for-linux-bug-can-wipe-all-your-user-files)

Reply:
Good catch.  Removed as it was unnecessary.